packages/pwa/app/pages/login/index.jsx

```diff
@@ -31,7 +31,8 @@ const Login = () => {
         } catch (error) {
             const message = /invalid credentials/i.test(error.message)
                 ? formatMessage({
-                      defaultMessage: 'Incorrect username or password, please try again.'
+                      defaultMessage: 'Incorrect username or password, please try again.',
+                      id: 'login.error.incorrect_username_password'
```

Comment:
```suggestion
                      id: 'login_page.error.incorrect_username_password'
```

Reply:
Is adding '_page' here because we want to differentiate it from the same-named Login component?

Reply:
yes, since we have a login as a component and login as a page. It would be better if we use `login_page` for this one